design/sketch.webidl

```diff
@@ -237,14 +237,6 @@ dictionary GPUBindGroupLayoutDescriptor {
 interface GPUBindGroupLayout {
 };
 
-// PipelineLayout
-dictionary GPUPipelineLayoutDescriptor {
-    sequence<GPUBindGroupLayout> bindGroupLayouts;
-};
-
-interface GPUPipelineLayout {
-};
-
```

Comment:
You should remove `createPipelineLayout` too. However, having a separate object can be helpful for signifying pipeline compatibility to the user.

Reply:
Removed `createPipelineLayout` too, thanks for the catch.

Pipeline layout wouldn't have been used for compatibility except for "bind group inheritance" if we choose to have that. The same inheritance can exist with pipelines so I'm not too worried.

Reply:
Sounds good to me